sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,254 @@
+- Partially implemented [`sycl_ext_intel_fpga_kernel_interface_properties`](doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc) extension [c5d6b89c]
+- Added support for `sycl::ext::oneapi::experimental::if_architecture_is` to 
+ESIMD kernels [7a811191]
+- Implemented `sycl::opencl::cl_*` aliases [e98d732406cc]
+- Implemented `annotated_arg` headers from [`sycl_ext_oneapi_annotated_arg`](doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc) extension [1649fe46]
+- Added `bfloat16` support to imf libdevice functions [378678a3] [e85b51fc] 
+[25a3813b] [2b00cf95]
+- Added initialization of `sycl::ext::oneapi::experimental::device_global` 
+without `device_image_scope` property [c4617790]
+- Implemented [`sycl_ext_oneapi_device_global`](doc/extensions/proposed/sycl_ext_oneapi_device_global.asciidoc)'s host-side memory operations [d6f5b352]
+- Added `sycl::device::get_info<info::device::aspects>` specialization 
+[7869c17a]
+- Added `rbegin()`, `rend()`, `crbegin()` and `crend()` methods to 
+`sycl::accessor`, `sycl::host_accessor` and `sycl::local_accessor` [baa3d739]
+- Added `accessor_ptr` alias to `sycl::accessor` and `sycl::local_accessor` 
+[6b4cd9d0]
+- Implemented `get_multi_ptr` method for `sycl::accessor` and 
+`sycl::local_accessor` [f4185c75]
+- Added `size_type` to `sycl::accessor` and `sycl::local_accessor` classes 
+[0a17e16c]
+- Implemented accessors implicit conversions [51b7f781]
+- Implemented templated `sycl::kernel_bundle::has_kernel` functions [50e798f9]
+- Implemented `sycl::kernel_bundle::get_kernel` function [cf018f16]
+- Added support for basic atomic operations for HIP AMD backend [c3c5e923]
+- Added the ability to set specialization constants in `sycl::compile` 
+[33cd6b1a]
+- Added the `value_type` and `binary_operation` member aliases and the 
+`dimensions` value to the `sycl::reducer` class [8fdc68d4]
+- Implemented `sycl::ext::intel::math::inv` free function [a90d0fd9]
+- Added APIs which emulate the behaviors of CUDA SIMD intrinsic [b0bfdfa7]
+- Added overloads on all binary operators with scalars as the left operand to 
+`sycl::marray`, and allowed `half`, `float`, `double` in `&&` and `||` operators 
+[a1787de5]
+- Added support of Intel(R) Xe Matrix Extensions with SIMD8 capability 
+[8316a9c8]
+- Implemented device headers part of `sycl::any_device_has` and 
+`sycl::all_devices_have` [81baa50b]
+- Added `host_task` in `sycl::target` enum class [a8401952]
+- Added support of `sycl::marray` to relational, common and some math functions 
+[c3a40760] [c5c7ac2e] [d8715418]
+- Implemented [`sycl_ext_oneapi_annotated_ptr`](doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc) extension [3250db55]
+- Implemented [`sycl_ext_oneapi_cuda_tex_cache_read`](doc/extensions/experimental/sycl_ext_oneapi_cuda_tex_cache_read.asciidoc) extension [5360825e650b]
+- Implemented `sycl::stream`'s size functions definition for device [368b6605]
```

Comment:
I would classify this as a bugfix. Something like "Fixed compilation issue when `sycl::stream::size` was called from device code"